### ISSUE TITLE
Order of parameters in `define_output`

### ISSUE THREAD

**Issue by mortendahl** [tf-encrypted/tf-encrypted#458]
Currently we do `tfe.define_output(player, logits, receive_output)` but perhaps `tfe.define_output(player, receive_output, logits)` would be better as it's closer to `receive_output(logits)`
